source/using-api-interface.rst

```diff
@@ -265,11 +265,11 @@ Let's send a message to dojot:
 
 If there is no output, the message was sent to MQTT broker.
 
-Note that we sent a message with the parameter ``-q 1``. This means that the message will use QoS 1,
+Note that we publish a message with the parameter ``-q 1``. This means that the message will use QoS 1,
 i.e., the message is guaranteed to be sent at least one time.
 
 
-**Also you can send a configuration message from dojot to the device to change some of its attributes.
```

Comment:
I think `send` was better

Reply:
Also you can send a configuration

Reply:
Changed for "Also you can send a..." RST and PO.